app.py

```diff
@@ -31,24 +40,24 @@ class ReviewLanguage(db.Model):
     name = db.Column(db.String(50), unique=True, nullable=False)
 
     def __repr__(self):
-        return '<User %r>' % self.id
+        return '<Language %r>' % self.name
 
 class ReviewRequest(db.Model):
     __tablename__ = 'review_request'
 
+    id = db.Column(db.Integer, primary_key=True)
     reviewer_id = db.Column(db.Integer, db.ForeignKey('reviewer.id'), nullable=False)
     review_language_id = db.Column(db.Integer, db.ForeignKey('review_language.id'), nullable=False)
     review_date = db.Column(db.DateTime, nullable=False, default=datetime.utcnow)
 
-    # It appears that SQLAlchemy requires a primary key...
     __table_args__ = (
-        db.PrimaryKeyConstraint('reviewer_id', 'review_date', 'review_language_id'),
-        # db.Index('reviewer_id'),
-        # db.Index('review_date'),
+        db.Index('review_date_ix', 'review_date'),
+        db.Index('reviewer_ix', 'reviewer_id', 'review_language_id'),
     )
 
     def __repr__(self):
-        return '<User %r>' % self.reviewer_id
+        return '<Review %r for Reviewer %r>' % (self.review_id, self.reviewer_id)
```

Comment:
Since we concurred on making the PK column be `id` (rather than `review_id`), we need to change the string substitution here as well -- `self.review_id` should be `self.id`.

Reply:
done